templates/web/angus/post_header_extra.html

```diff
@@ -0,0 +1,14 @@
+<div class="wrapper wrapper__breadcrumb">
```

Comment:
The use of .wrapper here is confusing to me. On a desktop map page for example, this now means that .wrapper__main_menu is shown, and so some special .nav-menu--breadcrumb display:none CSS has to be added (but OTOH .mappage .wrapper__breadcrumb is set display:none).
.wrapper is used to allow the navigation to move below/above at different widths; with a display:table. I'm not sure that it should be used for anything else.
The only thing I can see that it's doing is causing the floated menu to be contained, that is easily done with the cf class instead. Or please explain what else this is doing :)